packages/dashboard/src/components/widgets/tile/contants.ts

```diff
@@ -0,0 +1 @@
+export const delay = 3000;
```

Comment:
const are ALL CAPS and should denote the units 

`USE_CASE_DELAY_IN_SECS
`

Reply:
resolved